cmd/aries-js-worker/main.go

```diff
@@ -37,7 +37,7 @@ const (
 	ariesCommandPkg  = "aries"
 	ariesStartFn     = "Start"
 	ariesStopFn      = "Stop"
-	workers          = 2
+	workers          = 3
```

Comment:
2 was failing?

Reply:
I saw it on master branch

Reply:
latest PR on master failed, I will check it again.